packages/components/Notification/src/useNotification.ts

```diff
@@ -0,0 +1,12 @@
+import { NotificationProps } from './Notification.types';
+
+export const useNotification = (props: NotificationProps): NotificationProps => {
+  const { variant, startText, endText, ...rest } = props;
+
+  return {
+    variant,
+    startText,
+    endText,
+    ...rest,
+  };
+};
```

Comment:
Lol, a very not useful hook at the moment :D. I guess we might as well keep it because I think it will change.